control_apps/src/control_mngr.rs

```diff
@@ -11,6 +11,7 @@ use std::time::Duration;
 use log::{info, warn, debug};
 
 use crate::valve::Valve;
+use crate::measurement::Measurement;
```

Comment:
When I pull down this branch into my editor, my linter complains that `measurement` is an unresolved import.
```unresolved import `crate::measurement`

maybe a missing crate `measurement`?rustc(E0432)
control_mngr.rs(14, 12): maybe a missing crate `measurement`?```